docs/FormtronSchema.md

```diff
@@ -232,6 +233,7 @@ Here's an example.
     "tags": {
       "type": "array",
       "title": "Tag List",
+      "default": "",
```

Comment:
```suggestion
      "default": "New Item",
```

Reply:
In practice it makes more sense to just use an empty string though.